br/pkg/streamhelper/advancer_test.go

```diff
@@ -939,6 +939,9 @@ func TestSubscriptionPanic(t *testing.T) {
 
 	require.NoError(t, adv.OnTick(ctx))
 	failpoint.Enable("github.com/pingcap/tidb/br/pkg/streamhelper/subscription.listenOver.aboutToSend", "5*panic")
```

Comment:
you can try `testfailpoint.Enable`, it will use testing.Cleanup to defer the `failpoint.Disable`